### ISSUE TITLE
some question 

### ISSUE THREAD

**Issue by UestcJay** [2429581027/local-geometric-guided#2]
many thanks for your work! when evaluation on Scanned Data, The training set and test set are all from ref[46]? how to make the training and test set? looking forward to your reply.
